### ISSUE TITLE
Double `rules` Key when using `.bolt` Rules

### ISSUE THREAD

**Issue by mutewinter** [firebase/firebase-tools#105]
```
Firebase Tools 2.1.0
Firebase Bolt v0.5.0
```

Deploying via command line using `firebase deploy:rules` and specifying a `.bolt` rule file in my `firebase.json`. The rules that get set on my app contains a duplicate `rules` key, like so

```json
{
  "rules": {
    "rules": {
       // my correct rules here
    }
  }
}
```

**Comment by mutewinter** [comment]
I'm getting around this problem by using the JSON rules format in my `firebase.json` and adding an extra build step to generate the JSON rules before deploying them via the CLI.

**Comment by deltaepsilon** [comment]
+1

**Comment by drautb** [comment]
:pray:

**Comment by mbleigh** [comment]
Fixed in just-shipped 2.1.1! :shipit:
